vars/cloudFoundryDeploy.groovy

```diff
@@ -14,22 +15,109 @@ import groovy.transform.Field
 
 @Field Set STEP_CONFIG_KEYS = [
     'cloudFoundry',
-    'deployUser',
```

Comment:
why has this been removed?

Reply:
because it is only used on a log message, is this really necessary? It looks like it was used, but then changed and it only remains in the log.

Reply:
I can fix the tests if it can be removed.

Reply:
ok, your are right. `deployUser` is obsolete and can be removed ...